src/app/garfile/arrival/post.controller.js

```diff
@@ -19,38 +19,81 @@ module.exports = (req, res) => {
   const validationIds = ['arrivalPort'];
   const validationValues = [voyage.arrivalPort];
   const validationMsgs = ['Enter an arrival port'];
-  const validations = validator.genValidations(validator.notEmpty, validationIds, validationValues, validationMsgs);
 
   // Define port / date validation msgs
-  const portMsg = "If the Arrival port is 'ZZZZ' then you must also provide decimal coordinates to 4 decimal places";
+  const portMsg = 'If you do not have the airport code, enter \'ZZZZ\' and enter the latitude and longitude to 4 decimal places below.';
+  const portCodeMsg = 'The arrival airport code must be a minimum of 3 letters and a maximum of 4 letters.';
   const futureDateMsg = 'Arrival date must be today or in the future';
   const realDateMsg = 'Enter a real Arrival date';
   const timeMsg = 'Enter a real Arrival time';
+  const latitudeMsg = 'Value entered is incorrect. Please enter latitude to 4 decimal places.';
+  const longitudeMsg = 'Value entered is incorrect. Please enter longitude to 4 decimal places.';
```

Comment:
Remove the word please. GDS doesn't recommend you use it

Reply:
Should be done